packages/stateManager/mod_test.ts

```diff
@@ -27,7 +27,7 @@ import {
   type Tag,
 } from "./types.ts";
 
-async function setupTestManager() {
+export async function setupTestManager() {
```

Comment:
Move this to a test utils module if you want to reuse it in the other tests. *_test files should only be for the test runner to consume and should not have exports